lib/standards/aria-roles.js

```diff
@@ -119,7 +120,7 @@ const ariaRoles = {
     nameFromContent: true
   },
   combobox: {
-    type: 'widget',
+    type: 'composite',
```

Comment:
I think this should remain a widget. According to https://www.w3.org/WAI/ARIA/1.2/class-diagram/rdf_model.svg `combobox` is a composite, but so is a `spinbutton` and `spinbutton` is definitely a widget. So I think we're safe to say any interactable composite should be a widget.

```suggestion
    type: 'widget',
```

Reply:
Perhaps a good use of our 1:1 today is for us to talk about how you arrived at that conclusion, when there's two arrows coming off an arrow in the rdf model is makes it quite unclear.

Reply:
Add comment above `type` with `// Note: spec difference` as we want to keep `widget` role